src/configuration/mod.rs

```diff
@@ -1,56 +1,339 @@
 //! Configuration module
 
-mod prima;
-mod test;
+use crate::error::Error as PrimaDatadogError;
+use crate::TagTrackerConfiguration;
+use std::fmt::Display;
+use std::str::FromStr;
 
-pub use prima::{Config, Country, Environment};
-pub use test::TestConfiguration;
+/// By binding to 0.0.0.0:0 we're just letting the OS assign us a port, and letting anyone send us UDP packets on that port
```

Comment:
A bit more detail maybe? Feel free to reject

```suggestion
/// By binding to 0.0.0.0:0 we're just letting the OS assign us a port, and letting anyone send us UDP packets on that port
/// This is fine since the statsd protocol is unidirectional(only sends data) to the server
/// We can't just bind to a different address(eg. localhost) because that would prevent us from sending data to the server
```

Reply:
This is very welcome, thanks :pray: